BlackMarket/app/BlackMarketApp.swift

```diff
@@ -16,12 +16,15 @@ struct BlackMarketApp: App {
     WindowGroup {
       Group {
         if isLogged {
-          DashboardView(
-            store: Store(
-              initialState: DashboardDomain.State(),
-              reducer: DashboardDomain()
-            )
-          )
+          TabBarView(tabBarManager: TabBarManager(
+            routes: [
+              .home,
+              .purchases,
+              .favorites,
+              .shoppingCart,
+              .menu
+            ]
```

Comment:
As a rule of thumb, I won;t break lines if doesn't reach the line limit
```suggestion
            routes: [.home, .purchases, .favorites, .shoppingCart, .menu]
```